src/store.rs

```diff
@@ -0,0 +1,42 @@
+use std::path::{Path, PathBuf};
+use regex::Regex;
+
+const HASH_REG: &str = "^([[:alnum:]]+):([[:xdigit:]]+)$";
+
+pub fn hash_parse(hash: &str) -> (&str, &str) {
```

Comment:
Is it worth returning a struct rather than tuple, to improve readability? We might not do so much with these, that it matters?